apps/web/src/server/views/applications/components/status-tag.component.njk

```diff
@@ -0,0 +1,25 @@
+{% macro statusTag(status) %}
+	{% set statusTagColor = '' %}
+	{% set statusLabel = status %}
+
+	{% switch (status) %}
+	{% case 'Decision' %}
+		{% set statusTagColor = 'pink' %}
+	{% case 'Post decision' %}
+		{% set statusTagColor = 'grey' %}
+	{% case 'Pre-application' %}
+		{% set statusTagColor = 'green' %}
+	{% case 'Acceptance' %}
+		{% set statusTagColor = 'turquoise' %}
+	{% case 'Pre-examination' %}
+		{% set statusTagColor = 'blue' %}
+	{% case 'Examination' %}
+		{% set statusTagColor = 'purple' %}
+	{% case 'Recommendation' %}
+		{% set statusTagColor = 'yellow' %}
+	{% case 'Withdrawn' %}
+		{% set statusTagColor = 'orange' %}
+	{% endswitch %}
+
+	<strong class="govuk-tag govuk-tag--{{statusTagColor}}">{{ statusLabel }}</strong>
+{% endmacro %}
```

Comment:
Just checking, are we reusing these colours in the search results as well as the dashboards?

Reply:
yes we are